server/opts.go

```diff
@@ -1659,7 +1659,7 @@ func parseGateway(v interface{}, o *Options, errors *[]error, warnings *[]error)
 	return nil
 }
 
-var dynamicJSAccountLimits = JetStreamAccountLimits{-1, -1, -1, -1, 0, 0, false}
+var dynamicJSAccountLimits = JetStreamAccountLimits{-1, -1, -1, -1, 0, 0, 0, false}
```

Comment:
why does `max_memory` a byte limit default to `-1` while the `memory_max_stream_bytes` also a byte limit default to 0? 

Make using these quite challenging and easy to make mistakes

Reply:
this will work with -1 as well. however the value 0 does not mean it's disabled, just means not limited. 
If we had picked -1, then deployed jwt would have started to fail.

Reply:
@ripienaar I'll make sure they return `-1` to the client and make that independent of jwt. sounds good?